src/redisjson.rs

```diff
@@ -5,15 +5,21 @@
 // It can be operated on (e.g. INCR) and serialized back to JSON.
 
 use serde_json::Value;
-use jsonpath::Selector;
+use jsonpath_lib as jsonpath;
```

Comment:
Not sure I like this aliasing... why not just keep the original name and make it clear which crate is being used?